src/breedsDb.js

```diff
@@ -18,13 +18,63 @@ class BreedRepository {
         return this.breeds;
     }
 
+    set setBreeds(breedsCopy) {
+        this.breeds = breedsCopy;
+    }
+
+    findIndexById(id) {
+        const breedsCopy = this.getAllBreeds; 
+        const foundBreed = breedsCopy.findIndex((breed) => {
+            return breed.breedId === id;       
+        });
+        return foundBreed;  // foundBreed is either -1 or a valid index
```

Comment:
at first I thought this was a boolean variable but it's not - perhaps consider renaming